os/include/tinyara/mm/mm.h

```diff
@@ -783,6 +783,24 @@ void *zalloc_at(int heap_index, size_t size);
  * @endcond
  */
 
+#if defined(CONFIG_APP_BINARY_SEPARATION) && defined(__KERNEL__)
+
+#define MM_PART_FREE    0
+#define MM_PART_USED    1
+
+struct mm_partition_s {
+        uint32_t start;         /* Start address of the partition */
+        uint32_t size;          /* Size of the partition in bytes */
+        uint8_t status;         /* Current status of the partition, free / used */
+};
+
+/* Functions contained in mm_partition_mgr.c **************************************/
+void mm_initialize_partitions();
```

Comment:
You should fill argument type and name. That's our coding rules.
```
_partitions(void);
```

Reply:
There is a flash partition. Let's use a prefix `ram_` to avoid confusing.

Reply:
Ok Done